Command/DebugGatewayCommand.php

```diff
@@ -19,17 +19,22 @@ class DebugGatewayCommand extends Command implements ContainerAwareInterface
     use ContainerAwareTrait;
 
     protected static $defaultName = 'debug:payum:gateway';
+    protected Payum $payum;
+
+    public function __construct(Payum $payum)
+    {
+        $this->payum = $payum;
+        parent::__construct();
+    }
 
     /**
      * {@inheritDoc}
      */
-    protected function configure()
+    protected function configure(): void
     {
         $this
             ->setName(static::$defaultName)
-            ->setAliases(array(
```

Comment:
I think the alias must stay, it can be removed in the next major version.

Reply:
I did not remove that. I just switched to short-Array notation.

Please see Line 37 https://github.com/Payum/PayumBundle/pull/537/files/aa937259044ae5469d5182db8abea0afbb2b96e1#diff-d182f98c7ace14688eb49f452ff03d2270f2c005f9b8ee01eb245a7a88f069ff